spark/src/main/java/org/apache/iceberg/spark/SparkWriteOptions.java

```diff
@@ -29,4 +29,7 @@ private SparkWriteOptions() {
 
   // Overrides table property write.spark.fanout.enabled(default: false)
   public static final String FANOUT_ENABLED = "fanout-enabled";
+
+  // Fileformat for write operations
+  public static final String WRITE_FORMAT = "write-format";
```

Comment:
I feel like there are more write properties. Is there a reason we don't include others?